ui/lib/core/addon/components/form-field.hbs

```diff
@@ -133,12 +133,12 @@
       @value={{get @model this.valuePath}}
       @onChange={{this.setAndBroadcast}}
       @label={{this.labelString}}
-      @labelClass="title {{if (eq @mode 'create') 'is-5' 'is-4'}}"
+      @labelClass={{if @attr.options.isSectionHeader "title is-4" "is-label"}}
```

Comment:
Example of how this looks when `isSectionHeader: true`


<img width="1087" alt="Screenshot 2024-01-17 at 5 00 17 PM" src="https://github.com/hashicorp/vault/assets/68122737/931c9ae0-3f9d-4167-a758-ad9eb8aeea27">